### ISSUE TITLE
Several questions are "mu"

### ISSUE THREAD

**Issue by ec429** [8values/8values.github.io#79]
Some of the statements are just unclear on their premises — e.g. "International aid is a waste of money." I answered this as though it referred to aid sent and/or distributed by governments, since that's what most people mean by the term; but that's very different to saying that international _private charity_ is a waste of money.  Similarly, I read "Research should be conducted on an international scale." as being about international quangos like the IPCC — but multinational corporations also engage in research; the term 'conducted' is rather too vague (does it mean controlled, directed or performed?).

But then we get to problems like "The general populace makes poor decisions."  One can easily believe that individuals acting on their own account make good decisions, but that the aggregate decisions of 'the public will' are inevitably bad (the "public-choice theory" position), or vice-versa (the "wisdom-of-crowds" position).
"Regardless of political opinions, it is important to side with your country." is (intentionally?) vague about what is meant by "side with".  It's one thing to disapprove of, or even protest against, your country's foreign wars; it's quite another to actively sabotage your country in international negotiations, or to "talk down" your country's economy.

"The stronger the leadership, the better." isn't necessarily about government at all, and again is vague about what 'leadership' means; is it about having and communicating a coherent vision and direction, or is it about having control and _imposing_ that direction?
I'm not even sure what "Democracy is more than a decision-making process." is supposed to mean.
"Society was better many years ago than it is now." suffers from vagueness about what 'society' encompasses; public morals? interpersonal empathy? schemes of organisation?

There's a problem with "Same-sex marriage should be legal." in that it conflates two questions: should same-sex relationships be afforded the same consideration as opposite-sex relationships, and should legislation / the state be involved in what is, fundamentally, a private contract?  Bundling up the notions of the contract, religious traditions around it and special privileges under (e.g.) the tax code is exactly what makes this issue so divisive, because different sides (there are more than two) are fighting on different battlegrounds.

"We should open our borders to immigration." depends far too much on what other policies and practices are being pursued by the government; it is possible to believe immigration is a good thing while also believing it is incompatible with the social-democratic Bismarckian state, for instance.

"All people - regardless of factors like culture or sexuality - should be treated equally." leaves dangling the question 'by whom?'.  Requiring the Law and the State to treat all persons equally on the grounds that the State has unique and special powers of coercion is a far cry from giving the State the power to compel private individuals to treat others equally.

"It is important that we further my group's goals above all others." is vague about what kind of group is relevant.  If _ethnic_ or _cultural_ group, it's a fairly straightforward nationalism-versus-universalism question; but if you define your group by shared goals, then naturally you place that group's goals above others.

So in summary, there were a great many statements where I felt that both agreement and disagreement were wrong, while a neutral position fails to reflect my strong opinions on relevant subjects.  This may be why I, an an-cap, got https://8values.github.io/results.html?e=21.3&d=28.2&g=61.5&s=50.0 which doesn't really represent me well at all (I'm much more liberal than that on the Civil Axis, and not really as Patriotic as your quiz has me — though the Societal Axis does accurately reflect my willingness to let Chesterton's Fence temper Whiggish progressivism, balanced against a desire to see science unfettered to invent the future).

**Comment by GRBaset** [comment]
Yes, I agree that some questions are not very good. I'm pretty sure that "We should open our borders to immigration" and "All people - regardless of factors like culture or sexuality - should be treated equally" are mostly theoretical questions, though. It's what you'd choose for an ideal world in your view, without taking into account practical considerations.

**Comment by nissubjtsdjarwylwz** [comment]
The question "migrants benefits outweight the costs" is too narrow.  Migrants can be different. I do actively support skilled migrants to come to our country  and work on projects as we lack hands in some areas. But I do not support what's happening in Europe type migration.
